centml/cli/cluster.py

```diff
@@ -49,13 +80,19 @@ def ls(type):
 
 
 @click.command(help="Get deployment details")
+@click.argument("type", type=click.Choice(list(depl_type_map.keys())))
 @click.argument("id", type=int)
-def get(id):
-    deployment = api.get_inference(id)
+def get(type, id):
+    if type == platform_api_client.DeploymentType.INFERENCE:
+        deployment = api.get_inference(id)
+    elif type == platform_api_client.DeploymentType.COMPUTE:
+        deployment = api.get_compute(id)
+    else:
+        sys.exit("Please enter correct deployment type")
     state = api.get_status(id)
     ready_status = get_ready_status(deployment.status, state.service_status)
 
-    click.echo(f"Inference deployment #{id} is {ready_status}")
+    click.echo(f"Deployment #{id} is {ready_status}")
     click.echo(
         tabulate(
```

Comment:
In here, the Endpoint may not be https for compute deployment. We can just show `deployment.endpoint_url` no need to attach protocol.

Reply:
removed https from tabulate function